examples/external-data/src/mock-external-data-service/service.ts

```diff
@@ -239,6 +239,7 @@ export async function initializeExternalDataService(props: ServiceProps): Promis
 			}
 
 			// 3. Webhook exists, attempt to remove the subscriber from the webhook
+			// eslint-disable-next-line unicorn/no-negated-condition -- ignoring so the code doesn't have to be re-ordered
```

Comment:
@seanimam Would it be OK to reorder this code to remove the negated condition? The comments are numbered so I wasn't sure if re-ordering things would be OK.

Reply:
It's ok to reorder and just flip the 'a' and 'b'.